tests/inputs/HELIOTRON_iota

```diff
@@ -0,0 +1,46 @@
+# This is the DESC input file for a vacuum heliotron
```

Comment:
is this really vacuum? or just fixed iota?

Reply:
Not vacuum. Just fixed iota.